packages/@coorpacademy-components/src/template/common/dashboard/test/fixtures/with-learning-priorities.js

```diff
@@ -32,7 +32,8 @@ export default {
         title: 'Learning priorities',
         subtitle: 'Your top learning priorities defined by your company',
         type: 'learningPrioritiesCards',
-        key: 'learningPriorities'
+        key: 'learningPriorities',
+        cards: []
```

Comment:
why this modifications?

Reply:
Ah good catch, it's an unwanted change, reverted